api-gateway/content/nginx.conf

```diff
@@ -69,6 +70,19 @@ http {
             proxy_read_timeout   3600s;
             proxy_redirect       off;
         }
+		
+        location /iothubmanager-svc/ {
+            rewrite              /iothubmanager-svc/(.*) /$1 break;
+            proxy_pass           $iothubmanager_svc_endpoint;
+            proxy_pass_header    Authorization;
+            # TODO ~devis: remove - https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/issues/11
```

Comment:
Why is this comment here when the other 'location' chunks don't have this comment?